llvm/test/Transforms/LoopInterchange/refcost-overflow.ll

```diff
@@ -0,0 +1,44 @@
+; REQUIRES: asserts
+
+; RUN: opt < %s -passes=loop-interchange -S -debug 2>&1 | FileCheck %s
```

Comment:
test with -cache-line-size too to bolster checks?

Reply:
Don't think this is necessary.